nixos/doc/manual/configuration/x-windows.xml

```diff
@@ -96,6 +96,44 @@
   The options are named identically for all other display managers.
   </para>
  </simplesect>
+ <simplesect xml:id="sec-x11--graphics-cards-intel">
+  <title>Intel graphics drivers</title>
+  <para>
+   There are two choices for Intel graphics driver in X.org:
+   <literal>intel</literal> (provided by the package <package>xf86-video-intel</package>)
+   and <literal>modesetting</literal> (included in the <package>xorg-server</package> itself).
+   The default is <literal>modesetting</literal>.
+  </para>
+  <para>
+   The first driver is specific to Intel GPUs but is not recommended by most distribution:
+   it lacks several modern features (Glamor, 2D graphics acceleration via OpenGL, is not
+   possible with the latest DRI 3). The package was also last updated in 2015.
+   The second is a generic driver which uses the kernel
+   <link xlink:href="https://en.wikipedia.org/wiki/Mode_setting">mode setting</link>
+   (KMS) mechanism. This driver supports Glamor and is more actively maintained but may
+   perform worse in some cases (like in old chipsets).
+  </para>
+  <para>
+   The results vary depending on the hardware, so you may have to try both drivers.
+   You can set the driver as follows:
```

Comment:
This paragraph suggests how to use something else than the default, so it should probably be moved between the one explaining the default (`modesetting`) and specific alternatives (`intel`). You might just want to name the option that needs to be set to use something else, but not use not-yet-explained drivers (and put the specific example on how to use `intel` below the paragraph using it).

Reply:
Uhm, I'd rather keep the explanation of the drivers separate from the configuration. Anyway, I reorganized it a bit, better?